### ISSUE TITLE
download_profile_photo fails with "object ... has no attribute 'subclass_of_id'"

### ISSUE THREAD

**Issue by 3bl3gamer** [LonamiWebs/Telethon#284]
Telethon v0.15.0.3

`download_profile_photo(channel, filepath)` fails with
```
  File ".../telethon/telegram_client.py", line 572, in download_profile_photo
    if not isinstance(entity, TLObject) or type(entity).subclass_of_id in (
AttributeError: type object 'Channel' has no attribute 'subclass_of_id'
```
If this `subclass_of_id` is changed to `SUBCLASS_OF_ID` everything works fine.

Looks like that line was missed in https://github.com/LonamiWebs/Telethon/commit/afc4bd9cabacc80435e017af7a991bf9a0bf52e0

**Comment by Lonami** [comment]
Would you like to make a pull request? That way you're a step closer to the 4 you need for https://hacktoberfest.digitalocean.com/

**Comment by Lonami** [comment]
Fixed on #285.
